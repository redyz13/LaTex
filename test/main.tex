\documentclass{article}

\usepackage{graphicx}
\usepackage{amsmath}
\usepackage[plain]{algorithm}
\usepackage[noend]{algorithmic}
\usepackage[italian]{babel}
\usepackage{lipsum}
\usepackage{amsthm}
\usepackage{xcolor}
\usepackage[hidelinks]{hyperref}

\title{Paolo}
\author{Paolo}
\date{Maggio 2023}

\newtheorem{theorem}{Teorema}[section]

\renewcommand{\listalgorithmname}{Lista di algoritmi}
\floatname{algorithm}{Algoritmo}
\numberwithin{algorithm}{section}

\numberwithin{footnote}{section}
\renewcommand\thefootnote{\textcolor{red}{\arabic{footnote}}}

\begin{document}

\maketitle

\pdfbookmark[1]{\contentsname}{toc}

\tableofcontents

\cleardoublepage
\phantomsection
\addcontentsline{toc}{section}{\listalgorithmname}
\listofalgorithms

\include{src/section1}
\include{src/section2}

\end{document}
