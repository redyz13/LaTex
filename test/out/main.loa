\contentsline {algorithm}{\numberline {1.1}{\ignorespaces Ricerca binaria ricorsiva}}{3}{algorithm.1.1}%
\contentsline {algorithm}{\numberline {1.2}{\ignorespaces Ultima occorrenza}}{4}{algorithm.1.2}%
\contentsline {algorithm}{\numberline {2.1}{\ignorespaces M-Compute-Opt}}{5}{algorithm.2.1}%
