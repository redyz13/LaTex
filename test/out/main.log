This is pdfTeX, Version 3.141592653-2.6-1.40.25 (TeX Live 2023) (preloaded format=pdflatex 2023.5.24)  11 JUN 2023 21:35
entering extended mode
 restricted \write18 enabled.
 file:line:error style messages enabled.
 %&-line parsing enabled.
**d:/Workspace/Latex/test/main.tex
(d:/Workspace/Latex/test/main.tex
LaTeX2e <2022-11-01> patch level 1
L3 programming layer <2023-05-15> (d:/Tools/TinyTeX/texmf-dist/tex/latex/base/article.cls
Document Class: article 2022/07/02 v1.4n Standard LaTeX document class
(d:/Tools/TinyTeX/texmf-dist/tex/latex/base/size10.clo
File: size10.clo 2022/07/02 v1.4n Standard LaTeX file (size option)
)
\c@part=\count185
\c@section=\count186
\c@subsection=\count187
\c@subsubsection=\count188
\c@paragraph=\count189
\c@subparagraph=\count190
\c@figure=\count191
\c@table=\count192
\abovecaptionskip=\skip48
\belowcaptionskip=\skip49
\bibindent=\dimen140
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics/graphicx.sty
Package: graphicx 2021/09/16 v1.2d Enhanced LaTeX Graphics (DPC,SPQR)
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2022/05/29 v1.15 key=value parser (DPC)
\KV@toks@=\toks16
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics/graphics.sty
Package: graphics 2022/03/10 v1.4e Standard LaTeX Graphics (DPC,SPQR)
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics/trig.sty
Package: trig 2021/08/11 v1.11 sin cos tan (DPC)
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics-cfg/graphics.cfg
File: graphics.cfg 2016/06/04 v1.11 sample graphics configuration
)
Package graphics Info: Driver file: pdftex.def on input line 107.
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics-def/pdftex.def
File: pdftex.def 2022/09/22 v1.2b Graphics/color driver for pdftex
))
\Gin@req@height=\dimen141
\Gin@req@width=\dimen142
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/amsmath/amsmath.sty
Package: amsmath 2022/04/08 v2.17n AMS math features
\@mathmargin=\skip50

For additional information on amsmath, use the `?' option.
(d:/Tools/TinyTeX/texmf-dist/tex/latex/amsmath/amstext.sty
Package: amstext 2021/08/26 v2.01 AMS text
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/amsmath/amsgen.sty
File: amsgen.sty 1999/11/30 v2.0 generic functions
\@emptytoks=\toks17
\ex@=\dimen143
)) (d:/Tools/TinyTeX/texmf-dist/tex/latex/amsmath/amsbsy.sty
Package: amsbsy 1999/11/29 v1.2d Bold Symbols
\pmbraise@=\dimen144
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/amsmath/amsopn.sty
Package: amsopn 2022/04/08 v2.04 operator names
)
\inf@bad=\count193
LaTeX Info: Redefining \frac on input line 234.
\uproot@=\count194
\leftroot@=\count195
LaTeX Info: Redefining \overline on input line 399.
LaTeX Info: Redefining \colon on input line 410.
\classnum@=\count196
\DOTSCASE@=\count197
LaTeX Info: Redefining \ldots on input line 496.
LaTeX Info: Redefining \dots on input line 499.
LaTeX Info: Redefining \cdots on input line 620.
\Mathstrutbox@=\box51
\strutbox@=\box52
LaTeX Info: Redefining \big on input line 722.
LaTeX Info: Redefining \Big on input line 723.
LaTeX Info: Redefining \bigg on input line 724.
LaTeX Info: Redefining \Bigg on input line 725.
\big@size=\dimen145
LaTeX Font Info:    Redeclaring font encoding OML on input line 743.
LaTeX Font Info:    Redeclaring font encoding OMS on input line 744.
\macc@depth=\count198
LaTeX Info: Redefining \bmod on input line 905.
LaTeX Info: Redefining \pmod on input line 910.
LaTeX Info: Redefining \smash on input line 940.
LaTeX Info: Redefining \relbar on input line 970.
LaTeX Info: Redefining \Relbar on input line 971.
\c@MaxMatrixCols=\count199
\dotsspace@=\muskip16
\c@parentequation=\count266
\dspbrk@lvl=\count267
\tag@help=\toks18
\row@=\count268
\column@=\count269
\maxfields@=\count270
\andhelp@=\toks19
\eqnshift@=\dimen146
\alignsep@=\dimen147
\tagshift@=\dimen148
\tagwidth@=\dimen149
\totwidth@=\dimen150
\lineht@=\dimen151
\@envbody=\toks20
\multlinegap=\skip51
\multlinetaggap=\skip52
\mathdisplay@stack=\toks21
LaTeX Info: Redefining \[ on input line 2953.
LaTeX Info: Redefining \] on input line 2954.
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/algorithms/algorithm.sty
Package: algorithm 2009/08/24 v0.1 Document Style `algorithm' - floating environment
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/float/float.sty
Package: float 2001/11/08 v1.3d Float enhancements (AL)
\c@float@type=\count271
\float@exts=\toks22
\float@box=\box53
\@float@everytoks=\toks23
\@floatcapt=\box54
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/base/ifthen.sty
Package: ifthen 2022/04/13 v1.1d Standard LaTeX ifthen package (DPC)
)
\@float@every@algorithm=\toks24
\c@algorithm=\count272
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/algorithms/algorithmic.sty
Package: algorithmic 2009/08/24 v0.1 Document Style `algorithmic'
\c@ALC@unique=\count273
\c@ALC@line=\count274
\c@ALC@rem=\count275
\c@ALC@depth=\count276
\ALC@tlm=\skip53
\algorithmicindent=\skip54
) (d:/Tools/TinyTeX/texmf-dist/tex/generic/babel/babel.sty
Package: babel 2023/05/11 v3.89 The Babel package
\babel@savecnt=\count277
\U@D=\dimen152
\l@unhyphenated=\language8
 (d:/Tools/TinyTeX/texmf-dist/tex/generic/babel/txtbabel.def)
\bbl@readstream=\read2
\bbl@dirlevel=\count278
 (d:/Tools/TinyTeX/texmf-dist/tex/generic/babel-italian/italian.ldf
Language: italian 2022/03/27 v.1.4.07 Italian support for the babel system
\it@lettering=\count279
\it@doublequoteactive=\count280
\it@ISOcompliance=\count281
)) (d:/Tools/TinyTeX/texmf-dist/tex/generic/babel/locale/it/babel-italian.tex
Package babel Info: Importing font and identification data for italian
(babel)             from babel-it.ini. Reported on input line 11.
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/etoolbox/etoolbox.sty
Package: etoolbox 2020/10/05 v2.5k e-TeX tools for LaTeX (JAW)
\etb@tempcnta=\count282
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/lipsum/lipsum.sty (d:/Tools/TinyTeX/texmf-dist/tex/latex/l3packages/l3keys2e/l3keys2e.sty (d:/Tools/TinyTeX/texmf-dist/tex/latex/l3kernel/expl3.sty
Package: expl3 2023-05-15 L3 programming layer (loader) 
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def
File: l3backend-pdftex.def 2023-04-19 L3 backend support: PDF output (pdfTeX)
\l__color_backend_stack_int=\count283
\l__pdf_internal_box=\box55
))
Package: l3keys2e 2023-02-02 LaTeX2e option processing using LaTeX3 keys
)
Package: lipsum 2021-09-20 v2.7 150 paragraphs of Lorem Ipsum dummy text
\g__lipsum_par_int=\count284
\l__lipsum_a_int=\count285
\l__lipsum_b_int=\count286
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/lipsum/lipsum.ltd.tex)) (d:/Tools/TinyTeX/texmf-dist/tex/latex/amscls/amsthm.sty
Package: amsthm 2020/05/29 v2.20.6
\thm@style=\toks25
\thm@bodyfont=\toks26
\thm@headfont=\toks27
\thm@notefont=\toks28
\thm@headpunct=\toks29
\thm@preskip=\skip55
\thm@postskip=\skip56
\thm@headsep=\skip57
\dth@everypar=\toks30
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/xcolor/xcolor.sty
Package: xcolor 2022/06/12 v2.14 LaTeX color extensions (UK)
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics-cfg/color.cfg
File: color.cfg 2016/01/02 v1.6 sample color configuration
)
Package xcolor Info: Driver file: pdftex.def on input line 227.
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/graphics/mathcolor.ltx)
Package xcolor Info: Model `cmy' substituted by `cmy0' on input line 1353.
Package xcolor Info: Model `hsb' substituted by `rgb' on input line 1357.
Package xcolor Info: Model `RGB' extended on input line 1369.
Package xcolor Info: Model `HTML' substituted by `rgb' on input line 1371.
Package xcolor Info: Model `Hsb' substituted by `hsb' on input line 1372.
Package xcolor Info: Model `tHsb' substituted by `hsb' on input line 1373.
Package xcolor Info: Model `HSB' substituted by `hsb' on input line 1374.
Package xcolor Info: Model `Gray' substituted by `gray' on input line 1375.
Package xcolor Info: Model `wave' substituted by `hsb' on input line 1376.
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/hyperref/hyperref.sty
Package: hyperref 2023-04-22 v7.00x Hypertext links for LaTeX
 (d:/Tools/TinyTeX/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty
Package: ltxcmds 2020-05-10 v1.25 LaTeX kernel commands for general use (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/generic/iftex/iftex.sty
Package: iftex 2022/02/03 v1.0f TeX engine tests
) (d:/Tools/TinyTeX/texmf-dist/tex/generic/pdftexcmds/pdftexcmds.sty
Package: pdftexcmds 2020-06-27 v0.33 Utility functions of pdfTeX for LuaTeX (HO)
 (d:/Tools/TinyTeX/texmf-dist/tex/generic/infwarerr/infwarerr.sty
Package: infwarerr 2019/12/03 v1.5 Providing info/warning/error messages (HO)
)
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/kvsetkeys/kvsetkeys.sty
Package: kvsetkeys 2022-10-05 v1.19 Key value parser (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty
Package: kvdefinekeys 2019-12-19 v1.6 Define keys (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/generic/pdfescape/pdfescape.sty
Package: pdfescape 2019/12/09 v1.15 Implements pdfTeX's escape features (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/hycolor/hycolor.sty
Package: hycolor 2020-01-27 v1.10 Color options for hyperref/bookmark (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/letltxmacro/letltxmacro.sty
Package: letltxmacro 2019/12/03 v1.6 Let assignment for LaTeX macros (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/auxhook/auxhook.sty
Package: auxhook 2019-12-17 v1.6 Hooks for auxiliary files (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/latex/hyperref/nameref.sty
Package: nameref 2022-05-17 v2.50 Cross-referencing by name of section
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/refcount/refcount.sty
Package: refcount 2019/12/15 v3.6 Data extraction from label references (HO)
) (d:/Tools/TinyTeX/texmf-dist/tex/generic/gettitlestring/gettitlestring.sty
Package: gettitlestring 2019/12/15 v1.6 Cleanup title references (HO)
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/kvoptions/kvoptions.sty
Package: kvoptions 2022-06-15 v3.15 Key value format for package options (HO)
))
\c@section@level=\count287
)
\@linkdim=\dimen153
\Hy@linkcounter=\count288
\Hy@pagecounter=\count289
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2023-04-22 v7.00x Hyperref: PDFDocEncoding definition (HO)
Now handling font encoding PD1 ...
... no UTF-8 mapping file for font encoding PD1
) (d:/Tools/TinyTeX/texmf-dist/tex/generic/intcalc/intcalc.sty
Package: intcalc 2019/12/15 v1.3 Expandable calculations with integers (HO)
)
\Hy@SavedSpaceFactor=\count290
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/hyperref/puenc.def
File: puenc.def 2023-04-22 v7.00x Hyperref: PDF Unicode definition (HO)
Now handling font encoding PU ...
... no UTF-8 mapping file for font encoding PU
)
Package hyperref Info: Hyper figures OFF on input line 4182.
Package hyperref Info: Link nesting OFF on input line 4187.
Package hyperref Info: Hyper index ON on input line 4190.
Package hyperref Info: Plain pages OFF on input line 4197.
Package hyperref Info: Backreferencing OFF on input line 4202.
Package hyperref Info: Implicit mode ON; LaTeX internals redefined.
Package hyperref Info: Bookmarks ON on input line 4449.
\c@Hy@tempcnt=\count291
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/url/url.sty
\Urlmuskip=\muskip17
Package: url 2013/09/16  ver 3.4  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 4787.
\XeTeXLinkMargin=\dimen154
 (d:/Tools/TinyTeX/texmf-dist/tex/generic/bitset/bitset.sty
Package: bitset 2019/12/09 v1.3 Handle bit-vector datatype (HO)
 (d:/Tools/TinyTeX/texmf-dist/tex/generic/bigintcalc/bigintcalc.sty
Package: bigintcalc 2019/12/15 v1.5 Expandable calculations on big integers (HO)
))
\Fld@menulength=\count292
\Field@Width=\dimen155
\Fld@charsize=\dimen156
Package hyperref Info: Hyper figures OFF on input line 6066.
Package hyperref Info: Link nesting OFF on input line 6071.
Package hyperref Info: Hyper index ON on input line 6074.
Package hyperref Info: backreferencing OFF on input line 6081.
Package hyperref Info: Link coloring OFF on input line 6086.
Package hyperref Info: Link coloring with OCG OFF on input line 6091.
Package hyperref Info: PDF/A mode OFF on input line 6096.
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/base/atbegshi-ltx.sty
Package: atbegshi-ltx 2021/01/10 v1.0c Emulation of the original atbegshi
package with kernel methods
)
\Hy@abspage=\count293
\c@Item=\count294
\c@Hfootnote=\count295
)
Package hyperref Info: Driver (autodetected): hpdftex.
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2023-04-22 v7.00x Hyperref driver for pdfTeX
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/base/atveryend-ltx.sty
Package: atveryend-ltx 2020/08/19 v1.0a Emulation of the original atveryend package
with kernel methods
)
\Fld@listcount=\count296
\c@bookmark@seq@number=\count297
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/rerunfilecheck/rerunfilecheck.sty
Package: rerunfilecheck 2022-07-10 v1.10 Rerun checks for auxiliary files (HO)
 (d:/Tools/TinyTeX/texmf-dist/tex/generic/uniquecounter/uniquecounter.sty
Package: uniquecounter 2019/12/15 v1.4 Provide unlimited unique counter (HO)
)
Package uniquecounter Info: New unique counter `rerunfilecheck' on input line 285.
)
\Hy@SectionHShift=\skip58
)
\c@theorem=\count298
 (out/main.aux (out/src/section1.aux) (out/src/section2.aux))
\openout1 = `main.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for TS1/cmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for PU/pdf/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty
Package: epstopdf-base 2020-01-24 v2.11 Base part for package epstopdf
Package epstopdf-base Info: Redefining graphics rule for `.eps' on input line 485.
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
File: epstopdf-sys.cfg 2010/07/13 v1.3 Configuration of (r)epstopdf for TeX Live
))
LaTeX Info: Redefining \it@ocap on input line 26.
LaTeX Info: Redefining \it@ccap on input line 26.


File italian.ldf warning: 
italian.ldf    With OT1 encoding guillemets are poorly faked
italian.ldf    Use T1 encoding
italian.ldf    or specify a font with command \CaporaliFrom
italian.ldf    See the documentation concerning the babel-italian typesetting 
italian.ldf    on input line 26.

Package hyperref Info: Link coloring OFF on input line 26.
(out/main.out) (out/main.out)
\@outlinefile=\write3
\openout3 = `main.out'.

 (out/main.toc)
\tf@toc=\write4
\openout4 = `main.toc'.

 [1

{d:/Tools/TinyTeX/texmf-var/fonts/map/pdftex/updmap/pdftex.map}] (out/main.loa)
\tf@loa=\write5
\openout5 = `main.loa'.

 [2

]
\openout2 = `src/section1.aux'.

 (./src/section1.tex
Package hyperref Info: bookmark level for unknown algorithm defaults to 0 on input line 16.
 [3

] [4])
\openout2 = `src/section2.aux'.

 (./src/section2.tex
LaTeX Font Info:    Trying to load font information for OMS+cmr on input line 19.
 (d:/Tools/TinyTeX/texmf-dist/tex/latex/base/omscmr.fd
File: omscmr.fd 2022/07/10 v2.5l Standard LaTeX font definitions
)
LaTeX Font Info:    Font shape `OMS/cmr/m/n' in size <10> not available
(Font)              Font shape `OMS/cmsy/m/n' tried instead on input line 19.

Underfull \hbox (badness 10000) in paragraph at lines 43--45

 []


Overfull \hbox (1.49748pt too wide) in paragraph at lines 63--65
[]\OT1/cmr/m/n/10 M-Compute-Opt($\OML/cmm/m/it/10 j$\OT1/cmr/m/n/10 ) = $[](\OML/cmm/m/it/10 v[] \OT1/cmr/m/n/10 + [](\OML/cmm/m/it/10 p\OT1/cmr/m/n/10 (\OML/cmm/m/it/10 j\OT1/cmr/m/n/10 ))\OML/cmm/m/it/10 ;  []\OT1/cmr/m/n/10 (\OML/cmm/m/it/10 j \OMS/cmsy/m/n/10 ^^@
 []

[5



] [6]) (out/main.aux (out/src/section1.aux) (out/src/section2.aux))
Package rerunfilecheck Info: File `main.out' has not changed.
(rerunfilecheck)             Checksum: 259DBF61E077694CACBC76814A4E32FF;624.
 ) 
Here is how much of TeX's memory you used:
 11442 strings out of 478005
 183873 string characters out of 5838156
 1862845 words of memory out of 5000000
 31345 multiletter control sequences out of 15000+600000
 519750 words of font info for 57 fonts, out of 8000000 for 9000
 14 hyphenation exceptions out of 8191
 75i,12n,77p,565b,554s stack positions out of 10000i,1000n,20000p,200000b,200000s
 <d:/Tools/TinyTeX/texmf-var/fonts/pk/ljfour/jknappen/ec/tcrm1000.600pk><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx10.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx12.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmex10.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi10.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi7.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmr12.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmr17.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmr6.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmr7.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmr8.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmsy10.pfb><d:/Tools/TinyTeX/texmf-dist/fonts/type1/public/amsfonts/cm/cmti10.pfb>
Output written on out/main.pdf (6 pages, 156350 bytes).
PDF statistics:
 201 PDF objects out of 1000 (max. 8388607)
 164 compressed objects within 2 object streams
 62 named destinations out of 1000 (max. 500000)
 41 words of extra memory for PDF output out of 10000 (max. 10000000)

