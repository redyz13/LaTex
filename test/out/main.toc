\babel@toc {italian}{}\relax 
\babel@toc {italian}{}\relax 
\contentsline {section}{Lista di algoritmi}{2}{section*.2}%
\contentsline {section}{\numberline {1}Ricerca}{3}{section.1}%
\contentsline {subsection}{\numberline {1.1}Esempio di utilizzo}{4}{subsection.1.1}%
\contentsline {section}{\numberline {2}Interval Scheduling Pesato}{5}{section.2}%
\contentsline {paragraph}{Relazione di ricorrenza:}{5}{section*.7}%
