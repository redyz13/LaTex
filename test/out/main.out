\BOOKMARK [1][-]{toc.1}{\376\377\000I\000n\000d\000i\000c\000e}{}% 1
\BOOKMARK [1][-]{section*.2}{\376\377\000L\000i\000s\000t\000a\000\040\000d\000i\000\040\000a\000l\000g\000o\000r\000i\000t\000m\000i}{}% 2
\BOOKMARK [1][-]{section.1}{\376\377\000R\000i\000c\000e\000r\000c\000a}{}% 3
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000E\000s\000e\000m\000p\000i\000o\000\040\000d\000i\000\040\000u\000t\000i\000l\000i\000z\000z\000o}{section.1}% 4
\BOOKMARK [1][-]{section.2}{\376\377\000I\000n\000t\000e\000r\000v\000a\000l\000\040\000S\000c\000h\000e\000d\000u\000l\000i\000n\000g\000\040\000P\000e\000s\000a\000t\000o}{}% 5
